packages/renderer/src/lib/network/CreateNetwork.svelte

```diff
@@ -0,0 +1,321 @@
+  networkInfo.selectedProvider = selectedProvider;
+});
+
+// Auto-populate default subnet for ipvlan to help beginners
+$effect(() => {
+  // Only auto-populate when switching TO ipvlan from another driver
+  if (networkInfo.driver === 'ipvlan' && previousDriver !== 'ipvlan' && !networkInfo.subnet) {
+    // eslint-disable-next-line sonarjs/no-hardcoded-ip
+    networkInfo.subnet = '192.168.210.0/24';
+    // eslint-disable-next-line sonarjs/no-hardcoded-ip
+    networkInfo.gateway = '192.168.210.1';
+  }
+  previousDriver = networkInfo.driver;
+});
+
+let hasInvalidFields = $derived(!networkInfo.networkName || !networkInfo.selectedProvider);
+</script>
+
+<EngineFormPage
+  title="Create a Network"
+  inProgress={createNetworkInProgress}
+  showEmptyScreen={providerConnections.length === 0 && !createNetworkInProgress}>
+  {#snippet icon()}
+    <Icon icon={faNetworkWired} class="fa-2x" />
+  {/snippet}
+  {#snippet content()}
+    <div class="space-y-6">
+      <div hidden={createNetworkInProgress}>
+        <label for="networkName" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Name <span class="text-red-500">*</span></label>
+        <Input
+          bind:value={networkInfo.networkName}
+          name="networkName"
+          id="networkName"
+          placeholder="Network name"
+          required
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label
+          for="labels"
+          class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Labels (key=value):</label
+        >
+      
+        {#each networkInfo.labels as _, index (index)}
+          <div class="flex flex-col w-full py-1">
+            <div class="flex flex-row justify-center items-center w-full">
+              <Input
+                bind:value={networkInfo.labels[index]}
+                placeholder="e.g., env=prod or app=backend"
+                class="ml-2 w-full" />
+
+              <Button
+                type="link"
+                hidden={index === networkInfo.labels.length - 1}
+                on:click={(): void => deleteLabel(index)}
+                icon={faMinusCircle} />
+
+              <Button
+                type="link"
+                hidden={index < networkInfo.labels.length - 1}
+                on:click={addLabel}
+                icon={faPlusCircle} />
+            </div>
+          </div>
+        {/each}
+      </div>
+      
+
+      {#if providerConnections.length > 1}
+        <div hidden={createNetworkInProgress}>
+          <label for="providerChoice" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+            >Container engine</label>
+          <ContainerConnectionDropdown
+            id="providerChoice"
+            name="providerChoice"
+            bind:value={networkInfo.selectedProvider}
+            connections={providerConnections} />
+        </div>
+      {/if}
```

Comment:
issue(blocking): this need to be at the top of the form, since this define _where_ the network will be created (on which podman connection), this should be the first input.

<img width="1063" height="464" alt="Image" src="https://github.com/user-attachments/assets/7973be44-2558-438f-bf52-40dd32d5c80d" />